backend/internal/domain/helpers/tokens_generator.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"os"
+	"shodo/models"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+const (
+	AccessTokenLifeTime  = 3600
+	RefreshTokenLifeTime = 86400
+)
+
+var Secret = os.Getenv("JWT_SECRET")
```

Comment:
Mixing config and os env can be hard to track
Ideally, you would have a one place where you source everything, and wire it up in the main or some other function.

Reply:
Ok, thank you got it. I'll move it to main in next iteration of fixing the project :)